core/domain/acl_decorators.py

```diff
@@ -25,6 +25,23 @@
 current_user_services = models.Registry.import_current_user_services()
 
 
+def _is_activity_public(activity_rights):
+    """Checks whether activity is in public domain.
```

Comment:
No, I'm not sure about that. It's easier to deprecate things safely if all the public/publicized stuff is in one place.